### PR TITLE
Feature/issue 904 change to scss module admin components

### PR DESCRIPTION
**Pull request by MotrukOleg**
* [Github ticket](https://github.com/ita-social-projects/VictoryCenter-Back/issues/904)

## Description

Migrate all styles of components found in the /components/admin folder to CSS modules and thoroughly inspect the UI to see whether all the styles have migrated properly.

## How it looks

image

## Summary of change

changed all styles using incapsulated approach

## How to recreate changes

pull changes to your branch

## CHECK LIST
- [ ]  New tests was added or existing was modified
- [ ]  Changes has severe impact on users
- [ ]  Changes has medium impact on users
- [ ]  Changes has light impact on users
- [ ]  Test covetage was updated
- [ ]  PR meets all conventions

## Summary by CodeRabbit

* **Refactor**
  * Migrated many admin UI components to scoped CSS Modules for more consistent, isolated styling.

* **Style**
  * Minor formatting/whitespace tweaks and adjustments to SCSS variable imports.

* **Bug Fixes**
  * Added textarea blur handling to better reflect user interactions.

* **Tests**
  * Expanded unit tests for category selection, drag preview, and confirmation modal to cover many edge cases and interactions.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: src/components/admin/confirmation-modal/ConfirmationModal.test.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { render, screen, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
```
[the remark this answers is not in the document]
Comment (MykhailoKlapchuk) [reply src/components/admin/confirmation-modal/ConfirmationModal.test.tsx:1]: If you do not use it - delete

**Review by MykhailoKlapchuk — APPROVED** [review]
After this PR is merged we need to have a regression test of all apps because of its impact. You have a few comments from SonarQube and Rabbit. Check them.

File: src/components/admin/admin-panel-toolbar/AdminPageToolbar.module.scss
Comment (VolodymyrSushynskyi) [inline src/components/admin/admin-panel-toolbar/AdminPageToolbar.module.scss:1]: There is new line missing at the end of a file. Did you run `npm format` before committing?

**Review by VolodymyrSushynskyi — APPROVED** [review]
LGTM

File: src/components/admin/admin-panel-toolbar/AdminPageToolbar.module.scss
Comment (MotrukOleg) [reply src/components/admin/admin-panel-toolbar/AdminPageToolbar.module.scss:1]: I thought that linter showed me this mistake but it didn't.I will fix it

File: src/components/admin/button-tooltip/ButtonTooltip.module.scss [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
     &:hover,
     &:focus {
-        background-color: colors.$blue-50;
+        background-color: $blue-50;
```
Comment (snowflaki) [inline src/components/admin/button-tooltip/ButtonTooltip.module.scss]:
```suggestion
        background-color: c.$blue-50;
```

File: src/components/admin/button-tooltip/ButtonTooltip.module.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@use '../../../assets/sass/variables/colors';
+@import '../../../assets/sass/variables/colors.scss';
```
Comment (snowflaki) [inline src/components/admin/button-tooltip/ButtonTooltip.module.scss]:
```suggestion
@use '../../../assets/sass/variables/colors' as c;
```

File: src/components/admin/button/Button.module.scss [hunk 4]
```diff
@@ -1,11 +1,11 @@
-@use '../../../assets/sass/variables/colors';
+@import '../../../assets/sass/variables/colors.scss';
```
Comment (snowflaki) [inline src/components/admin/button/Button.module.scss]: Please replace `@import` with `@use`. The Sass team has officially deprecated `@import` because it pollutes the global namespace and can lead to variable collisions and slower compilation times

**Review by L1ght1234 — APPROVED** [review]
